### PR TITLE
Remove elm css

### PR DESCRIPTION
**Pull request by fbentz**

### CODE REVIEW

**Comment by Natim** [comment]
What about the styles that you removed? Shall we put them in a CSS file instead?

**Comment by fbentz** [comment]
I'll redo the styles with just a minimal css in a next PR. Feel free to use your own css preproccessor or something like that. ;)

**Comment by Natim** [comment]
I don't understand why you want to do it in another PR.

As far as I'm concerned, merging this PR will break the current status of master.

I understand that the goal of elm-kitchen is not to provide a way to handle CSS, however I feel that it would still be a good idea to propose a way to manage them.

**Comment by fbentz** [comment]
Do you have an idea about that?

**Comment by Natim** [comment]
Other than putting a main.css file in the public folder? Nope.

**Comment by fbentz** [comment]
That's what I had in mind.

File: skeleton/src/Views/Page.elm [hunk 1]
```diff
@@ -25,41 +23,12 @@ frame : Config -> ( String, List (Html msg) ) -> Document msg
...
     { title = title ++ " | elm-kitchen"
     , body =
-        [ div []
-            [ defaultCss
-            , viewHeader config
-            , div [ css [ padding2 (Css.em 1) zero ] ] content
-            ]
-            |> toUnstyled
+        [ viewHeader config
+        , main_ [] content
```
Comment (n1k0) [inline skeleton/src/Views/Page.elm]: Nit: we could simply have `body = viewHeader config :: content` here.

**Review by n1k0 — APPROVED** [review]
LGTM. I'm gonna push a commit removing the obsolete npm post install command though.

File: skeleton/src/Views/Page.elm (hunk 1 above)
Comment (n1k0) [reply skeleton/src/Views/Page.elm]: Sorry I pushed the change for this, it's up to you really.

**Comment by n1k0** [comment]
Could you please do a release? I think it's a major one regarding the changes.

**Comment by Natim** [comment]
That's awesome thanks !

**Comment by fbentz** [comment]
Done v4.0.0

**Pull request merged**